client/src/constants/form_choices.json

```diff
@@ -0,0 +1,43 @@
+{
+  "concentration": {
+    "main": {
+      "Bullock01": "Bullock (2001) Physical Form",
+      "Bullock01Power": "Bullock (2001) Power-Law",
+      "Duffy08": "Duffy (2008) Power-Law",
+      "Zehavi11": "Zehavi (2011) Power-Law",
+      "Ludlow16": "Ludlow (2016)",
+      "Ludlow16Empirical": "Ludlow (2016) Empirical"
+    },
+    "additional": {
+      "F": 0.01,
+      "K": 3.4,
+      "a": 9.0,
+      "b": -0.13,
+      "c": 1.0,
+      "ms": 2000000000000.0,
+      "sample": {
+        "relaxed": "relaxed",
+        "full": "full"
+      },
+      "f": 0.02,
+      "C": 650,
+      "c0_0": 3.395,
+      "c0_z": -0.215,
+      "beta_0": 0.307,
+      "beta_z": 0.54,
+      "gamma1_0": 0.628,
+      "gamma1_z": -0.047,
+      "gamma2_0": 0.317,
+      "gamma2_z": -0.893
+    }
+  },
```

Comment:
These are just copies of values in backendconstants! Draw from that and they actually have the structure you need, already, instead of you having to marshal things in manually.